test/mason/mason-test/test/sampletest2.chpl

```diff
@@ -0,0 +1,6 @@
+
+proc test() {
```

Comment:
Can we add usage of the `FizzBuzz` library to demonstrate it is accessible when compiled?  Maybe define a simple function in `FizzBuzz` and call it here.

Reply:
see latest commit, should give enough usage examples of both using a dependency and using your own library in testing.